.github/workflows/ko-build-tag.yaml

```
@@ -17,15 +17,21 @@ permissions:
     # via OIDC token exchange (LITELLM_BASE_URL and LITELLM_MODEL are
     # pinned in the called workflow's env: block), so there are no
     # GitHub repo secrets to map at the call site. id-token: write is
     # required here because a reusable workflow cannot elevate
     # permissions beyond what its caller grants. LFXV2-2019.
+    #
+    # TEMPORARY: continue-on-error is set to true while LFXV2-2134 (structured
+    # JSON output + retry in the LiteLLM adapter) is being fixed. The eval still
+    # runs and its result is visible, but a failure will not block the release.
+    # Remove continue-on-error once LFXV2-2134 is resolved.
     name: Weekly-Brief Live Eval (release gate)
+    continue-on-error: true
```

actionlint (1.7.12)

[error] when a reusable workflow is called with "uses", "continue-on-error" is not available. only following keys are allowed: "name", "uses", "with", "secrets", "needs", "if", and "permissions" in job "eval-live"

(syntax-check)